encoders/firebase-encoders-processor/firebase-encoders-processor.gradle

```diff
@@ -20,21 +20,21 @@ def jvm = org.gradle.internal.jvm.Jvm.current()
 
 
 dependencies {
-    implementation project(':encoders:firebase-encoders-processor:test-support')
+    implementation project(':encoders:firebase-encoders')
     implementation 'com.google.auto.service:auto-service-annotations:1.0-rc6'
-    implementation 'com.squareup:javapoet:1.11.1'
+    implementation 'com.squareup:javapoet:1.13.0'
     compileOnly 'javax.annotation:javax.annotation-api:1.3.2'
     implementation 'com.google.guava:guava:28.1-jre'
 
-    implementation "com.google.auto.value:auto-value-annotations:1.6.5"
```

Comment:
would it make sense to make these a pan-project var, like protobuf ver? In a follow up change I mean.